spec/openapi.yaml

```diff
@@ -325,6 +325,19 @@ tags:
       platform and specify additional filtering criteria to narrow down data
       returned by the reports.
 
+  - name: Attachments
+    description: |-
+      The Attachments API allows upload, download and link binary data with entity.
+      Unlinked attachments will be deleted after 2 days of the last upload
+      Client is responsible for entity uid validation and removing attachments within entity removal process
+      The following attachment types are supported:
+      - **jobs**: attachements for jobs
+      - **issues**: attachments for issues
+      - **strings**: attachments for strings
```

Comment:
I think we should add some short detaisl on how entity ids are constructed for the various domains?
@AlexArcher and @andreyfurdylo can you confirm those formulas for strings, jobs and issues?

Reply:
job uid - 12 random alpha-numeric (a-z0-9) characters.

Reply:
for issue: {accountUid}-{issueUid}
for issue comment: {accountUid}-{issueUid}-{commentUid}
Examples:
8f14e45fc-0b0535d1d34a
8f14e45fc-0b0535d1d34a-c82010dd3bca